corehq/apps/data_interfaces/models.py

```diff
@@ -1162,23 +1162,34 @@ def _handle_case_duplicate_new(self, case, rule, process_updates=False):
             existing_duplicate = None
 
         current_hash = CaseDuplicateNew.case_and_action_to_hash(case, self)
+        case_became_closed = not self.include_closed and case.closed
```

Comment:
I did verify that it was possible to modify closed cases, but for a rule that doesn't include closed cases, a false modification like this will only lead to unnecessary processing. The only thing "recently closed" cases will do is delete its previous entry. So, in the rare event that a closed case was modified, the only thing that would happen is that we would attempt to delete the duplicate for it, which should have already been deleted when the case was originally closed. I would like to clean this up in the future, however